lib/routemaster/middleware/error_handling.rb

```diff
@@ -11,6 +11,7 @@ class ErrorHandling < Faraday::Response::Middleware
         (404..404) => Errors::ResourceNotFound,
         (405..405) => Errors::MethodNotAllowed,
         (409..409) => Errors::ConflictResource,
+        (410..410) => Errors::ResourceGone,
```

Comment:
non-blocking:

unfortunate that we aren't using `[X]` instead of `X..X`. there's only 1 range here that isn't `X..X`. :confused: 

perhaps a cleanup task for a future PR.